File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        StringBuilder sb = new StringBuilder();
        
        for(int i=0; i<my_string.length(); i++) {
            char ch = my_string.charAt(i);
            
            if(Character.isLowerCase(ch)) {
                sb.append(Character.toUpperCase(ch));
            } else {
                sb.append(Character.toLowerCase(ch));
            }
        }
        
        return sb.toString();
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.js
function solution(my_string) {
    let answer = "";
    
    for(let i=0; i<my_string.length; i++) {
        let char = my_string[i];
        
        if(char === char.toLowerCase()) {
            answer += char.toUpperCase();
        } else {
            answer += char.toLowerCase();
        }
    }
    
    return answer;
    
//     return [...my_string]
//            .map(char => char === char.toLowerCase()
//                 ? char.toUpperCase() 
//                 : char.toLowerCase())                                
//             .join('');
    
//     return my_string.split('')
//            .map(char => char === char.toLowerCase()
//                 ? char.toUpperCase()
//                 : char.toLowerCase())
//            .join('');
}